Tests/ReadyCampaignDispatcherSpec.swift

```diff
@@ -21,6 +21,17 @@ class ReadyCampaignDispatcherSpec: QuickSpec {
                                                 permissionService: permissionService,
                                                 campaignRepository: campaignRepository)
                 dispatcher.delegate = delegate
+
+                dispatcher.httpSession = URLSessionMock.mock(originalInstance: .shared)
```

Comment:
hmm. does that work without calling `URLSessionMock.startMockingURLSession()`?

Reply:
Yes, but only because `CampaignDispatcher.httpSession` wasn't private and I was setting it directly from the test. I'll change it to private and just swizzle it out like you did.